src/sgx/CMakeLists.txt

```diff
@@ -1,103 +1,156 @@
-#
-# Created by Joshua Heinemann on 18.06.20.
-# TU-Braunschweig (heineman@ibr.cs.tu-bs.de)
-#
-#set(RUNNER_LIBS
-#        faaslet_lib
-#        wasm
-#        wavmmodule
-#        util
-#        module_cache
-#        system
-#        )
+
+# NASM configuration
 enable_language(ASM_NASM)
 set(CMAKE_ASM_NASM_FLAGS -felf64)
 set(CMAKE_ASM_NASM_COMPILE_OBJECT "<CMAKE_ASM_NASM_COMPILER> <INCLUDES> ${CMAKE_ASM_NASM_FLAGS} -o <OBJECT> <SOURCE>")
+
+# SGX config
 set(SGX_DEBUG_MODE 1)
 set(SGX_SDK_PATH /opt/intel/sgxsdk)
 set(SGX_SDK_LIB_PATH ${SGX_SDK_PATH}/lib64)
 set(SGX_SDK_ENCLAVE_SIGNER ${SGX_SDK_PATH}/bin/x64/sgx_sign)
 set(SGX_SDK_ENCLAVE_EDGER8R ${SGX_SDK_PATH}/bin/x64/sgx_edger8r)
 set(SGX_C_GLOBAL_FLAGS -m64)
 set(THREADS_PREFER_PTHREAD_FLAG ON)
+
+# WAMR-related config
 set(SGX_WAMR_SOURCE_FILES SGXWAMRWasmModule.cpp sgx_wamr_native_symbols.cpp)
 file(GLOB SGX_WAMR_ASM_SOURCE_FILES SGXWAMRWasmModule.S)
 set(SGX_WAMR_ENCLAVE_SOURCE_FILES sgx_wamr_enclave.cpp sgx_wamr_native_symbols_wrapper.cpp)
 set(SGX_WAMR_URTS_OBJECTS ${CMAKE_CURRENT_BINARY_DIR}/sgx_wamr_enclave_u.o)
 set(SGX_WAMR_ENCLAVE_TRTS_OBJECTS ${CMAKE_CURRENT_BINARY_DIR}/sgx_wamr_enclave_t.o)
 set_source_files_properties(${SGX_WAMR_ASM_SOURCE_FILES} PROPERTIES LANGUAGE ASM_NASM)
-if(FAASM_WAMR_SUPPORT)
-message ("-- SGX-WAMR Configuration:
+
+if (FAASM_WAMR_SUPPORT)
+    # Debug message
+    message("-- SGX-WAMR Configuration:
      FAASM_SGX_WAMR_HEAP_SIZE: ${FAASM_SGX_WAMR_HEAP_SIZE}KB
      FAASM_SGX_WAMR_MODULE_ERROR_BUFFER_SIZE: ${FAASM_SGX_WAMR_MODULE_ERROR_BUFFER_SIZE}B
      FAASM_SGX_WAMR_INSTANCE_DEFAULT_HEAP_SIZE: ${FAASM_SGX_WAMR_INSTANCE_DEFAULT_HEAP_SIZE}B
      FAASM_SGX_WAMR_INSTANCE_DEFAULT_STACK_SIZE: ${FAASM_SGX_WAMR_INSTANCE_DEFAULT_STACK_SIZE}B")
-    add_definitions(-DFAASM_SGX_WAMR_HEAP_SIZE=${FAASM_SGX_WAMR_HEAP_SIZE} -DFAASM_SGX_WAMR_MODULE_ERROR_BUFFER_SIZE=${FAASM_SGX_WAMR_MODULE_ERROR_BUFFER_SIZE}
-            -DFAASM_SGX_WAMR_INSTANCE_DEFAULT_HEAP_SIZE=${FAASM_SGX_WAMR_INSTANCE_DEFAULT_HEAP_SIZE} -DFAASM_SGX_WAMR_INSTANCE_DEFAULT_STACK_SIZE=${FAASM_SGX_WAMR_INSTANCE_DEFAULT_STACK_SIZE})
-endif()
+
+    add_definitions(
+            -DFAASM_SGX_WAMR_HEAP_SIZE=${FAASM_SGX_WAMR_HEAP_SIZE}
```

Comment:
Just more newlines here, without them it was a wall of text and parameters. When it's set up like that it's hard to read, and also hard to spot diffs when someone makes changes.